### PR TITLE
Create AOCODAF405V3 target

### PR DESCRIPTION
**Pull request by lida2003**

### CODE REVIEW

**Review by nerdCopter — APPROVED** [review]
- approving, did not deeply inspect.

File: configs/AOCODAF405V3/config.h [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    TIMER_PIN_MAP( 9, PB1 , 2,  0)
+
+#define ADC_INSTANCE ADC3
+#define ADC3_DMA_OPT        1
+
+#define GPS_UART SERIAL_PORT_USART1
+#define SERIALRX_UART SERIAL_PORT_USART2
+#define MSP_UART SERIAL_PORT_UART5
+
+#define DEFAULT_FEATURES                (FEATURE_OSD | FEATURE_LED_STRIP | FEATURE_GPS)
```
Comment (haslinghuis) [inline configs/AOCODAF405V3/config.h:112]: DEFAULT_FEATURES are already defined in cloud build.

File: configs/AOCODAF405V3/config.h [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+#define MAX7456_SPI_INSTANCE SPI2
+#define PINIO1_BOX 40
+#define PINIO2_BOX 41
+#define PINIO3_BOX 42
+#define PINIO1_CONFIG 129
+#define PINIO2_CONFIG 129
+#define PINIO3_CONFIG 1
+#define FLASH_SPI_INSTANCE SPI3
+#define SERIALRX_PROVIDER SERIALRX_CRSF
+#define USE_SPI_GYRO
```
Comment (haslinghuis) [inline configs/AOCODAF405V3/config.h:133]: USE_SPI_GYRO is already defined in target.h

File: configs/AOCODAF405V3/config.h [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+#define DEFAULT_CURRENT_METER_SCALE 206
+#define BEEPER_INVERTED
+#define SYSTEM_HSE_MHZ 8
+#define MAX7456_SPI_INSTANCE SPI2
+#define PINIO1_BOX 40
+#define PINIO2_BOX 41
+#define PINIO3_BOX 42
+#define PINIO1_CONFIG 129
+#define PINIO2_CONFIG 129
+#define PINIO3_CONFIG 1
```
Comment (haslinghuis) [inline configs/AOCODAF405V3/config.h:130]: 0x01 is default - so this can be removed.

File: configs/AOCODAF405V3/config.h [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+#define USE_ACC
+#define USE_ACC_SPI_MPU6000
+#define USE_ACC_SPI_ICM42688P
+#define USE_BARO
+#define USE_BARO_DPS310
+#define USE_BARO_BMP280
+#define USE_BARO_MS5611
+#define USE_MAG
+#define USE_MAG_HMC5883
+#define USE_MAG_QMC5883
```
Comment (haslinghuis) [inline configs/AOCODAF405V3/config.h:41]: Do not include MAG drivers unless physical on board.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**